java/restheart/pom.xml

```diff
@@ -10,7 +10,7 @@
     <mongodb.version>4.2</mongodb.version>
     <maven.compiler.source>11</maven.compiler.source>
     <maven.compiler.target>11</maven.compiler.target>
-    <restheart.version>[5.1.6,5.2)</restheart.version>
```

Comment:
could you use `[5.1,5.2)` ? no need to bump each patch level version

Reply:
done